<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>606</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Meiryo UI</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame_2{
background-color:#fff;
border-radius:2px;
border-color:#e0e0e0;
border-style:solid;
border-width:1px;
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="CustomPushButton" name="FindLibraryButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="toolTip">
          <string>search image preset</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover
{
	background-color:#cdcdcd;
	border-radius:3px;
}

QPushButton:pressed
{
	background-color:#c0c0c0;
	border-radius:3px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="rc.qrc">
           <normaloff>:/uiIcon/src/ui/resource/icon/search.png</normaloff>:/uiIcon/src/ui/resource/icon/search.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LibraryPath">
         <property name="font">
          <font>
           <family>Yu Gothic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
border: 1px solid transparent;
border-bottom-color:#E0E0E0;
border-radius:0px;</string>
         </property>
         <property name="placeholderText">
          <string>search .fdsi or input .fdsi path </string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="CustomPushButton" name="LoadLibraryButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="toolTip">
          <string>preset open</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover
{
	background-color:#cdcdcd;
	border-radius:3px;
}

QPushButton:pressed
{
	background-color:#c0c0c0;
	border-radius:3px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="rc.qrc">
           <normaloff>:/uiIcon/src/ui/resource/icon/ic_import_contacts_black_24dp.png</normaloff>:/uiIcon/src/ui/resource/icon/ic_import_contacts_black_24dp.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QFrame" name="ListFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>1200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustIgnored</enum>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="GraphicFrame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>250</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4"/>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="CustomPushButton" name="RemoveButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>36</width>
                 <height>36</height>
                </size>
               </property>
               <property name="toolTip">
                <string>remove selected Item</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton:hover
{
	background-color:#cdcdcd;
	border-radius:3px;
}

QPushButton:pressed
{
	background-color:#c0c0c0;
	border-radius:3px;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="rc.qrc">
                 <normaloff>:/uiIcon/src/ui/resource/icon/ic_delete_black_24dp.png</normaloff>:/uiIcon/src/ui/resource/icon/ic_delete_black_24dp.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>36</width>
                 <height>36</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame{
background-color:#fff;
border-radius:2px;
border-color:#e0e0e0;
border-style:solid;
border-width:1px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="CustomPushButton" name="FindImageButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="toolTip">
          <string>search image</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover
{
	background-color:#cdcdcd;
	border-radius:3px;
}

QPushButton:pressed
{
	background-color:#c0c0c0;
	border-radius:3px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="rc.qrc">
           <normaloff>:/uiIcon/src/ui/resource/icon/search.png</normaloff>:/uiIcon/src/ui/resource/icon/search.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ImagePath">
         <property name="font">
          <font>
           <family>Yu Gothic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
border: 1px solid transparent;
border-bottom-color:#E0E0E0;
border-radius:0px;</string>
         </property>
         <property name="placeholderText">
          <string>search images or input images path or D&amp;D</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="CustomPushButton" name="LoadImageButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="toolTip">
          <string>import</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover
{
	background-color:#cdcdcd;
	border-radius:3px;
}

QPushButton:pressed
{
	background-color:#c0c0c0;
	border-radius:3px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="rc.qrc">
           <normaloff>:/uiIcon/src/ui/resource/icon/Import.png</normaloff>:/uiIcon/src/ui/resource/icon/Import.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_6">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="CustomPushButton" name="FinishEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>96</width>
           <height>48</height>
          </size>
         </property>
         <property name="toolTip">
          <string>edit finish</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color:#1976D2;
	border-width:1px;
	border-radius:3px;
	color:white;
}
QPushButton:hover
{
	background-color:#176CBF;
	border-width:1px;
	border-radius:3px;
	color:white;
}
QPushButton:pressed
{
	background-color:#1565B3;
	border-width:1px;
	border-radius:3px;
	color:white;
}
QPushButton:disabled
{
background-color:silver;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="rc.qrc">
           <normaloff>:/uiIcon/src/ui/resource/icon/ic_save_white_24dp.png</normaloff>:/uiIcon/src/ui/resource/icon/ic_save_white_24dp.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomPushButton</class>
   <extends>QPushButton</extends>
   <header>src/ui/widget/customPushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
